dev-ruby/excon/excon-0.82.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26"
+
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+RUBY_FAKEGEM_EXTRADOC="README.md changelog.txt"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
```

Comment:
Tests fail for me and it looks like you are missing a number of test dependencies in the ebuild.

Also, there are additional tests in the "tests" directory that are not being run now.